modules/nixos/holo-deploy.nix

```diff
@@ -10,4 +14,16 @@
       description = "IP addres or host name to connect to the host";
     };
   };
+
+  config = {
+    environment.etc."nix/sourceInfo.json" = {
+      enable = true;
+      text = builtins.toJSON (builtins.removeAttrs self.sourceInfo ["outPath"]);
+    };
+
+    environment.etc."nix/source" = {
+      enable = true;
+      text = self.sourceInfo.outPath;
```

Comment:
this will be a text file that contains the path to the sources stored underneath `/nix/store/...`